src/index.js

```diff
@@ -31,6 +31,9 @@ function setup () {
   if (settings.sql)
     db = require('./database/sequelize')
 
+  if (settings.embedEmoji)
+    tt = settings.embedEmoji
+
```

Comment:
Since we are getting rid of `tt`, We could change this to

```javascript
if (!settings.embedEmoji)
   settings.embedEmoji = '⭐️'
```

or use reactionEmoji as default

```javascript
if (!settings.embedEmoji)
   settings.embedEmoji = settings.reactionEmoji
```

Reply:
I had considered using reactonEmoji as the default, only issue was that if a custom emoji is used it might look a little weird since the name might not be pluralised properly.

Reply:
Added this as :star:

Additionally a thought I had was that using :star: would be better for consistency, if someone updates the bot but doesn't update their config then it will continue to use :star: rather than suddenly changing to something different.
The default should be visible in the config to new users so they can change it if desired.

Reply:
Right, agreed. If only custom emojis worked in the footer. That, `@username` links in the header, and bots with Twitch emotes. One can dream